.clangd

```diff
@@ -0,0 +1,4 @@
+# https://clangd.llvm.org/config
+
+CompileFlags:
+  Remove: -fconcepts-diagnostics-depth=10
```

Comment:
why is this needed? What is the effect?

Reply:
Yes. About "**why this is needed**":  some guys would like to use clangd, a popular Language Protocol Server provided by llvm, to do "code completion", "go to definition" and so on. So I added this config file.
    About "**what is the effect**":  The "-fconcepts-diagnostics-depth" option is provided by gcc not by clang compiler. However, clangd is based on clang compiler. So I removed this flag to suppress clangd's warning.
    More details: https://clangd.llvm.org/